components/script/dom/bindings/codegen/Bindings.conf

```diff
@@ -488,6 +488,22 @@ DOMInterfaces = {
     'canGc': ['Abort', 'GetResponseXML', 'Response', 'Send'],
 },
 
+'XPathEvaluator': {
+    'weakReferenceable': True,
+},
+
+'XPathExpression': {
+    'weakReferenceable': True,
+},
+
+'XPathNSResolver': {
+    'weakReferenceable': True,
+},
+
+'XPathResult': {
+    'weakReferenceable': True,
+},
+
```

Comment:
I wasn't quite sure what to put here, so this might be all wrong.

Reply:
If you don't actually use weak reference functionality, you don't need to make any changes to this file.

Reply:
Removed it!